draft-ysl-cats-metric-definition.md

```diff
@@ -148,34 +147,102 @@ Figure 1 shows the logic of metrics in Level 0, Level 1, and Level 2.
                          +-----------------+---------------+
                          |                 |               |
                      +---+----+        +---+----+      +---+----+
-         L1 Metrics: |  M1-1  |        |  M1-2  |      |  M1-3  | (...)
+         L1 Metrics: |  M1-1  |        |  M1-2  |      |  M1-3  | ...
                      +---^----+        +---^----+      +----^---+
                          |                 |                |
               +--------+-+-------+       +-+-------+        |
               |        |         |       |         |        |
            +--+---+ +--+---+ +---+--+ +--+---+ +---+--+  +--+---+
-L0 Metrics:| M0-1 | | M0-2 | | M0-3 | | M0-4 | | M0-5 |  | M0-6 | (...)
+L0 Metrics:| M0-1 | | M0-2 | | M0-3 | | M0-4 | | M0-5 |  | M0-6 | ...
            +------+ +------+ +------+ +------+ +------+  +------+
 
 ~~~
-{: #fig-cats-metric title="Logic of CATS Metrics in levels"}
-
+{: #fig-metric-levels title="Logic of CATS Metrics in levels"}
 
 
 # Representation of Metrics
 
-This section includes the detailed representation of metrics. [RFC9439] gives a good way to show the representation of some network metrics which is used for network capabilities exposure to applications. This document further describes the representation of CATS metrics.
+The representation of metrics is a key component of the CATS architecture. It defines how metrics are encoded and transmitted over the network. The representation should be flexible enough to accommodate different types of metrics and their associated units and precisions.
+
+This section includes the detailed representation of the CATS metrics. The design follows similar principles used in other similar IETF specifications, such as the network performance metrics defined in {{?RFC9439}}.
+
+## Definition of CATS Metric
+
+A CATS metric is represented using a set of fields, each describing a property of the metric. The following fields are introduced:
+
+<!-- JRG Note and TODO: Define each of the types, formats, etc.. Do we need to standardize them? -->
```

Comment:
The fields and examples below looks good to me.  
For the question on whether to standardize them, it looks like L1 and L2 metrics also need these fields for representation.  So maybe we should standardize them, And also it relates to where the normalization should be implemented. If it's implemented in the CATS domain, then these fields should be recognized and propagated.  Look for more feedback.